### PR TITLE
Made datetime attribute insert and match time-zone invariant

### PR DESCRIPTION
**Pull request by shiladitya-mukherjee**
## What is the goal of this PR?

Enables the BDD tests to check timezone-invariance of inserting and reading datetime attributes.

## What are the changes implemented in this PR?

- Added "set time-zone is: {time_zone_label}" which sets environment timezone to time_zone_label.
- Fixed datetime attribute parsing by bounding the number of splits of the attribute by ":" to 2.

### CODE REVIEW

File: test/behaviour/util/UtilSteps.java
Comment (krishnangovindraj) [inline test/behaviour/util/UtilSteps.java:28]: Missing new line at the end of the file

File: .bazelrc
Comment (krishnangovindraj) [inline .bazelrc:23]: I don't think this line was part of the ARM build fix. Please verify and remove.

**Review by krishnangovindraj — CHANGES_REQUESTED** [review]
Couple of small comments

We still need to wait for the M2 build fix to be merged before we merge this , OR rebase it to strip out those changes.

File: .bazelrc
Comment (flyingsilverfin) [reply .bazelrc:23]: I think we actually want this :) let's keep it

Comment (krishnangovindraj) [reply .bazelrc:23]: Is this line alone valid syntax, or does it have to be:
`common --java_runtime_version=remotejdk_11`

File: .bazelrc
Comment (flyingsilverfin) [inline .bazelrc:22]: is there a 'common' block?

Comment (flyingsilverfin) [reply .bazelrc:22]: try putting it first. Move `--incompatible_strict_action_env` there as well if it works, then delete `run` since it'l be empty

Comment (shiladitya-mukherjee) [reply .bazelrc:22]: `Inherited 'common' options: --java_runtime_version=remotejdk_11 --incompatible_strict_action_env
ERROR: --java_runtime_version=remotejdk_11 :: Unrecognized option: --java_runtime_version=remotejdk_11`

**Review by flyingsilverfin — APPROVED** [review]

**Comment by flyingsilverfin** [comment]
Looks good! Please rebase onto the latest development and we can merge :)

**Comment by shiladitya-mukherjee** [comment]
> Looks good! Please rebase onto the latest development and we can merge :)

done!

**Pull request merged**
